docs/couchbase_columnar_api/errors.rst

```diff
@@ -0,0 +1,29 @@
+==============
+Errors
+==============
+
+.. contents::
+    :local:
+
+.. module:: couchbase_columnar.exceptions
+
+
+ColumnarError
+++++++++++++++++++++++++++++++++
+.. autoclass:: ColumnarError
+
+InvalidCredentialError
+++++++++++++++++++++++++++++++++
+.. autoclass:: InvalidCredentialError
+
+QueryError
+++++++++++++++++++++++++++++++++
+.. autoclass:: QueryError
```

Comment:
It appears that the `server_message` and `code` properties are missing from the generated HTML docs

Reply:
Updated 👍